Xbox release post (#1362)
jellyfin/jellyfin.org: blog/2025/04-15-XboxUWP-0.9/index.mdx

---
title: Jellyfin for Xbox 0.9.0
authors:
  - JPVenson
slug: xbox-v0.9.0
tags: [release, xbox, uwp]
---

> Better late then never. The UWP wrapper for jellyfin has a new Maintainer and its first release since 2020.

Reviewer: ```suggestion
Better late then never. The UWP app for Jellyfin now has a new Maintainer and its first release in 5 years.
```

> Better late then never. The UWP app for Jellyfin now has a new Maintainer and its first release in 5 years.

Reviewer 2: ```suggestion
Better late then never. The UWP app for Jellyfin now has a new maintainer and its first release in 5 years.
```

> Better late then never. The UWP app for Jellyfin now has a new maintainer and its first release in 5 years.

Reviewer 3: ```suggestion
Better late then never. The Xbox app for Jellyfin now has a new maintainer and its first release in 5 years.
```

Don't call it uwp

/- JPVenson

{/* truncate */}

The Jellyfin app on Xbox has been unmaintained for quite a while. 

> While it was on lifesupport for the last couple of years, we were occasionally merging PRs and fixing bugs.

Reviewer 3: > we were occasionally merging PRs and fixing bugs.

Bit too technical, I'd omit the PR part and write it more like

> we were occasionally fixing bugs.

> With the last release now 5 years old, and the continued interest in Jellyfin being available on Xbox overall, I decided to take over at least the superficial duties of keeping it operational and releasing if there are new Features or bug fixes.

Reviewer 2: ```suggestion
With the last release now 5 years old, and the continued interest in Jellyfin being available on Xbox overall, I decided to take over at least the superficial duties of keeping it operational and releasing when there are new features or bug fixes.
```

> The Jellyfin app on Xbox has been unmaintained for quite a while. 
> While it was on lifesupport for the last couple of years, we were occasually merging PRs and fixing bugs though the last release is now over 5 years ago.
> 
> As there is still continued intrest in the application and for Jellyfin to be available on Xbox overall, i decided to take over at least the superficial duties of keeping it operational and releasing if there are new Features or Bugfixes.

Reviewer: ```suggestion
The Jellyfin app on Xbox has been unmaintained for quite a while. 
While it was on lifesupport for the last couple of years, we were occasionally merging PRs and fixing bugs.

With the last release now 5 years old, and the continued interest in Jellyfin being available on Xbox overall, I decided to take over at least the superficial duties of keeping it operational and releasing if there are new Features or bug fixes.
```

# UWP 0.9.0 release

> This new release has now been approved and is available as a new version in the microsoft store for up-to-date Xbox consoles.

Reviewer: ```suggestion
This new release has now been approved and is available as a new version on the Microsoft store for up-to-date Xbox consoles.
```

> It contains a number of bug fixes, but the main update is the move to WebView2. 

Reviewer 3: > move to WebView2.

More like -->

> move to a modern webbrowser implementation.

Although I still think that can be a bit too technical, it should be easier to understand. The "webview" term is not known to non-developers and especially if you reference a specific .NET API..

Reviewer 3: ChatGPT suggests something like this

"We upgraded the app's web browser engine to a faster, more modern one."

> It generally contains a number of bugfixes but namely also the Update to WebView2. 

Reviewer: ```suggestion
It contains a number of bug fixes, but the main update is the move to WebView2. 
```

> As the app is nothing more then a web browser with a few tweaks, this is a huge step up and allows Xboxes to use a Chromium-based browser instead of the old Legacy-Edge one. 

Reviewer 2: ```suggestion
As the app is nothing more than a web browser with a few tweaks, this is a huge step up and allows the Xbox to use a Chromium-based browser instead of the old, legacy Edge one. 
```

> As the app is nothing more than a web browser with a few tweaks, this is a huge step up and allows the Xbox to use a Chromium-based browser instead of the old, legacy Edge one. 

Reviewer 3: > allows the Xbox to use a Chromium-based browser instead of the old, legacy Edge one.

More like "to use a modern browser implementation that is more secure and supports newer features for a better experience" or something like that

> As the app is nothing more then an Web-Browser with a few tweeks, this is a huge step up and allows xboxs to use a chromium based browser instead of the old Legacy-Edge one. 

Reviewer: ```suggestion
As the app is nothing more then a web browser with a few tweaks, this is a huge step up and allows Xboxes to use a Chromium-based browser instead of the old Legacy-Edge one. 
```

> This brings for example the capabilities to display HVEC encoded media alongside a few other features that makes the whole experience quite a bit better.

Reviewer: ```suggestion
This brings the capability to display HEVC encoded media alongside a few other features that makes the whole experience a lot better.
```

# What to expect

I will be actively managing pull requests and try my best with bug reports, though we are still very reliant on your help with new features and bugfixes. 

> If there are enough features and bug fixes done and merged, you can however expect more releases in the future although i will not give a definitive timeline for that.

Reviewer: ```suggestion
If there are enough features and bug fixes done and merged, you can expect more releases in the future, though I will not give a definitive timeline for that.
```
